### PR TITLE
fix title and virtual devices number

### PR DESCRIPTION
**Pull request by AnkRaiza**
Relates #1017 #1018

### CODE REVIEW

File: src/pages/MyAccountPage.tsx [hunk 1]
```diff
@@ -90,6 +95,7 @@ export class MyAccountPage extends React.Component<MyAccountPageProps, MyAccount
     }
 
     async componentDidMount() {
+        this.props.setPageSubtitle("My account");
```
Comment (dmarvp) [inline src/pages/MyAccountPage.tsx]: "My Account"

File: src/services/subscription-plan.ts [hunk 2]
```diff
@@ -38,13 +38,13 @@ export class Payment {
...
-            return userDetail.silentEchoToken ? 1 : 0;
-        }
-
-        userDetail = await auth.currentUserDetails();
-        if (userDetail) {
-            return userDetail.silentEchoToken ? 1 : 0;
+        let details = userDetail;
+        if (!details) details = await auth.currentUserDetails();
+        if (details) {
+            const virtualDevicesLength = process.env.NODE_ENV === "production"
```
Comment (dmarvp) [inline src/services/subscription-plan.ts]: I think we don't need the "env" check here. Either they have the virtualDevices property or they have the silentEchoToken :)

Comment (AnkRaiza) [reply src/services/subscription-plan.ts]: they could have both and this check is because so far we have the virtual devices on dev only

Comment (dmarvp) [reply src/services/subscription-plan.ts]: but if they have both then the ones that matter are the ones from the virtualDevices property right?

**Pull request merged**